production/helm/loki/values.yaml

```diff
@@ -1906,7 +1907,8 @@ distributor:
   # -- Node selector for distributor pods
   nodeSelector: {}
   # -- Tolerations for distributor pods
-  tolerations: []
+  tolerations:
```

Comment:
Is this formatting change intentional? Putting the value on a new line is used inconsistently across this file (see `nodeSelector` above). It would probably make sense to do it one way or the other. Or just leave these lines untouched :)